book/06-github/sections/1-setting-up-account.asc

```diff
@@ -77,13 +77,13 @@ In <<_add_email_addresses>> we can see some of the different states that are pos
 The top address is verified and set as the primary address, meaning that is where you'll get any notifications and receipts.
 The second address is verified and so can be set as the primary if you wish to switch them.
 The final address is unverified, meaning that you can't make it your primary address.
-If GitHub sees any of these in commit messages in any repository on the site, it will be linked to your user now.
+If GitHub sees any of these in commit messages in any repository on the site, it can now be linked to your user.
```

Comment:
I probably just screwed up the GIT stuff.  I'm just now learning about
GIT.  I did get something confused with github as I have a username
(42degrees) and an e-mail address (sg_github@pingbot.com) which I created
some time back but didn't have the time then (or maybe the desire) to fully
learn git.  Sorry if it was confusing.

On Fri, Aug 5, 2016 at 4:04 PM, Ben Straub <notifications@github.com> wrote:

> In book/06-github/sections/1-setting-up-account.asc
> <https://github.com/progit/progit2/pull/630#discussion_r73766080>:
>
> > @@ -77,13 +77,13 @@ In <<_add_email_addresses>> we can see some of the different states that are pos
> >  The top address is verified and set as the primary address, meaning that is where you'll get any notifications and receipts.
> >  The second address is verified and so can be set as the primary if you wish to switch them.
> >  The final address is unverified, meaning that you can't make it your primary address.
> > -If GitHub sees any of these in commit messages in any repository on the site, it will be linked to your user now.
> > +If GitHub sees any of these in commit messages in any repository on the site, it can now be linked to your user.
>
> Relatedly: apparently you authored some of these commits with a different
> email address which isn't currently linked to your account. 😄
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/progit/progit2/pull/630/files/2cb2f4600180163db5ac70f1f440e28cbf88c975#r73766080>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABmTbi_4DMIZmqToy59T02TLEDhw0jZLks5qc7NlgaJpZM4JYCxK>
> .
>